src/plugins/intel_cpu/tests/unit/streams_info_table_test.cpp

```diff
@@ -1404,4 +1404,378 @@ INSTANTIATE_TEST_SUITE_P(StreamsInfoTable,
                                          _1sockets_ecores_tput_5,
                                          _1sockets_mock_tput_1));
 
+struct StreamGenerateionTestCase {
+    int input_stream;
+    int input_thread;
+    int input_request;
+    int input_model_prefer;
+    ov::hint::SchedulingCoreType input_type;
+    bool input_ht_value;
+    bool input_ht_changed;
+    bool input_cpu_value;
+    bool input_cpu_changed;
+    ov::hint::PerformanceMode input_pm_hint;
+    ov::threading::IStreamsExecutor::ThreadBindingType input_binding_type;
+    std::vector<std::vector<int>> input_proc_type_table;
+    bool output_ht_value;
```

Comment:
please add output check for core type and performance hint

Reply:
updated.